plugins/outputs/influxdb/README.md

```diff
@@ -39,7 +39,7 @@ The InfluxDB output plugin writes metrics to the [InfluxDB v1.x] HTTP or UDP ser
   ## tag is not set the 'retention_policy' option is used as the default.
   # retention_policy_tag = ""
 
-  ## If true, the 'retention_policy_tag' will not be removed from the metric.
+  ## If true, the 'retention_policy_tag' will not be added to the metric.
```

Comment:
Since the retention policy tag already exists on the metric, I think we should change this to "If true, ... will be removed from"